package.json

```diff
@@ -32,10 +32,8 @@
         "package": "npm run package -w packages/toolkit -w packages/amazonq",
         "newChange": "echo 'Must specify subproject/workspace with -w packages/<subproject>' && false",
         "createRelease": "echo 'Must specify subproject/workspace with -w packages/<subproject>' && false",
-        "format": "prettier --check plugins && npm run format -w packages/ --if-present",
-        "formatfix": "prettier --write plugins && npm run formatfix -w packages/ --if-present",
-        "lint": "npm run lint -w packages/ --if-present && npm run format",
-        "lintfix": "eslint -c .eslintrc.js --fix --ext .ts packages plugins && npm run formatfix",
+        "lint": "npm run lint -w packages/ --if-present",
+        "lintfix": "eslint -c .eslintrc.js --ignore-path .gitignore --ignore-pattern '**/*.json' --ignore-pattern '**/*.gen.ts' --ignore-pattern '**/types/*.d.ts' --ignore-pattern '**/src/testFixtures/**' --ignore-pattern '**/resources/js/graphStateMachine.js' --fix --ext .ts packages plugins",
```

Comment:
Future: need to extract the eslint invocation to something like `scripts/lint.ts`.